### PR TITLE
Miscellaneous minor fixes

### PR DESCRIPTION
**Pull request by lucaswerkmeister**
See individual commit messages.

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
CI is failing. As far as I can tell, the install script requires `$MYSQL_USER` and `$MYSQL_PASS` to be set, but I don’t see where that happens? I’m confused.

**Comment by AlaaSarhan** [comment]
yeah that's my fault ... I will fix that

**Comment by lucaswerkmeister** [comment]
I was going to extract the fix commits into a separate pull request :/

File: .travis.install.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ ln -s "$originalDirectory" mediawiki-term-store
 
 cd mediawiki-term-store
 
-composer install
+composer instal
```
Comment (lucaswerkmeister) [inline .travis.install.sh]: uh

**Pull request merged**